recipes-domu/domu-image-android/domu-image-android.bb

```diff
@@ -44,9 +44,12 @@ update_local_conf() {
 
     cd ${S}
 
-    # override what xt-distro wants as machine as we will only use
-    # machines understood by Poky
-    base_update_conf_value ${local_conf} MACHINE "qemux86-64"
+    # Android uses its own machine for its build, but we need to define
```

Comment:
It is not clear what is this name and where it comes from and what is the problem with the existing build and why Android's machine name was not used before.

Reply:
I see two reasons to override setting defined in xt-images:
* paths created during build will have names related to product/platform.
* it's not good when common code overrides values set in product specific configs. In general case, it could be better if common code provides only default values.

Reply:
Please put some more description in the comment